app/src/ActionModal/components/DelegationModal.vue

```diff
@@ -154,7 +154,7 @@
       />
       <TmFormMsg
         v-else-if="$v.amount.$error && !$v.amount.min"
-        :min="smallestAmount"
+        :min="isPolkadotController ? 0 : smallestAmount"
```

Comment:
why isn't smallestAmount===0 for a controller account?

Reply:
Refactor done.